src/server/contentSecurityPolicy.js

```diff
@@ -158,6 +158,7 @@ export default {
       'https://www.gstatic.com',
       'https://*.hotjar.com',
       ' data:',
+      'http://localhost:3001',
```

Comment:
Hvorfor trenges den her?

Reply:
Godt spørsmål! Var for at bildene ikke var tillatt fra localhost, men det ser ikke ut til å være et problem nå når det lastes fra public-folderen.